src/components/AppSidebar/AppSidebar.vue

```diff
@@ -398,7 +414,9 @@ export default {
 
 	data() {
 		return {
+			changeTitleTranslated: t('Change title'),
 			closeTranslated: t('Close'),
+			favoriteTranslated: t('Favorite'),
```

Comment:
Couldn't this be declared where it's used?

Reply:
Same for `changeTitleTranslated`

Reply:
The extractor library does not support this https://github.com/lukasgeiter/gettext-extractor/issues/40 which is why it must be used this way in this library https://github.com/nextcloud/nextcloud-vue/blob/2ac55083188ff5ebb4a3defee052f2508b4b165e/src/components/EmojiPicker/EmojiPicker.vue#L180-L194